web/utils.js

```diff
@@ -60,7 +60,7 @@ function domReadyMast() {
         }
     }
     if (document.annotate) {
-        $('a[class=r]').tooltip({ left: 5, showURL: false });
+        $('a.r').tooltip({ left: 5, showURL: false });
```

Comment:
$('a[class=r]') failed to create a tooltip on link with class="r version_color_n"